torch/utils/file_baton.py

```diff
@@ -32,16 +32,24 @@ def try_acquire(self):
         except FileExistsError:
             return False
 
-    def wait(self):
+    def wait(self, show_lock_path=False):
```

Comment:
```suggestion
    def wait(self, warn_after_seconds=None):
```

Reply:
Also, any thoughts on whether this should be specified at construction or passed in to the wait argument?